DragaliaAPI.Database/Entities/DbPlayerDragonReliability.cs

```diff
@@ -53,5 +54,11 @@ public static DbPlayerDragonReliability Create(string deviceAccountId, Dragons i
             GetTime = DateTimeOffset.UtcNow,
             LastContactTime = DateTimeOffset.UtcNow
         };
+        if (DragonConstants.fullBondDragons.Contains(id))
+        {
+            newReliability.Exp = DragonConstants.bondXpLimits[^1];
+            newReliability.Level = (byte)DragonConstants.bondXpLimits.Length;
+        }
+        return newReliability;
```

Comment:
Can you change this to use the masterasset DefaultReliabilityLevel and remove the fullbonddragonslist